### DISCUSSION TITLE
Push-Benachrichtigung beim HMIP-PSM

### DISCUSSION THREAD

**Discussion by gerd2005** [thkl/hap-homematic#603] [Q&A] [answered]
Hallo,

ich habe eine Regel in der CCU gebaut, mit der die Steckdose PSM an- und ausgeschaltet wird.
Nun möchte ich beim Ein- und Ausschalten jedes mal eine Push-Benachrichtigung erhalten.

Hat jemand eine Idee, wie man das am besten bauen kann?

Danke und lieben Gruß
Gerd

**Comment by blackbasket** [comment]
Hi @gerd2005 

CloudMatic Push Service oder Pushover verwenden... HomeKit unterstützt nur integrierte Push-Nachrichten... bisher für Türen und Fenster...

LG,
Marcel

**Comment by gerd2005** [comment]
Hallo @blackbasket 
danke für den Hinweis. Ich dachte eher daran mit virtuellen Geräten oder so zu arbeiten. 
Die Steckdose bzw. vielmehr die Regel triggert ein virtuelles Gerät, welches dann wiederum eine Push-Nachricht auslöst. Gibt es etwas?

**Comment by blackbasket** [comment, marked as answer]
Dazu müsstest du das virtuelle Gerät als Fenster/Tür in HomeKit definieren... dann sollte es gehen... das müsstest du aber selbst testen, da habe ich zurzeit keine Gelegenheit zu...

**Comment by gerd2005** [comment]
Das probiere ich mal, danke!
